mmaction/apis/train.py

```diff
@@ -154,3 +161,65 @@ def train_model(model,
     if cfg.omnisource:
         runner_kwargs = dict(train_ratio=train_ratio)
     runner.run(data_loaders, cfg.workflow, cfg.total_epochs, **runner_kwargs)
+
+    if test['test_last'] or test['test_best']:
```

Comment:
Actually, https://github.com/open-mmlab/mmaction2/blob/master/mmaction/core/evaluation/eval_hooks.py#L225, we can use `save_best= cfg.get('evaluation', {}).get('save_best', False)` as an indicator to judge whether to test the best ckpt, `runner.meta['hook_msgs']['best_ckpt']` or `eval_hook.best_ckpt_path` can be use as the path of the best ckpt.

Reply:
For the first suggestion, I think `save_best` doesn't necessarily mean the user also wants to test the best (during validation) checkpoint, so I let users specify it explicitly. 
For the second one, I think that makes sense, I will use `eval_hook.best_ckpt_path` and check if it exists.

Reply:
Agree, if the test case is the same as the validation case, it is indeed no need to test the best one again.